decompiler/astdump.py

```diff
@@ -274,10 +273,10 @@ def ind(self, diff_indent=0, ast=None):
         # shouldn't indent in case there's only one or zero objects in this object to print
         if ast is None or len(ast) > 1:
             self.indent += diff_indent
-            self.p(u'\n' + self.indentation * self.indent)
+            self.p('\n' + self.indentation * self.indent)
 
     def p(self, string):
         # write the string to the stream
-        string = unicode(string)
+        string = str(string)
```

Comment:
Do we in py3 need this line?

Reply:
Did some tests debug print in there and it was as thought always <class 'str'>, so i think i will test a while without the line before i commit it.